### PR TITLE
misc. updates

### PR DESCRIPTION
**Pull request by jorgeorpinel**
- Misc. copy edits

### CODE REVIEW

File: content/docs/command-reference/metrics/show.md [hunk 1]
```diff
@@ -42,9 +42,9 @@ compares them with a previous version.
   the workspace. Note that both options can be combined, for example using the
   `-aT` flag.
 
-- `--all-commits` - the same as `-a` or `-T` above, but applies to _all_ Git  
-  commits as well as the workspace. Useful for printing metric file contents for
-  the entire existing commit history of the project.
+- `--all-commits` - same as `-a` or `-T` above, but applies to _all_ Git commits
+  as well as the workspace. Useful for downloading all the data used in the
```
Comment (shcheklein) [inline content/docs/command-reference/metrics/show.md:46]: why downloading?

File: content/docs/command-reference/push.md [hunk 2]
```diff
@@ -71,9 +71,9 @@ in the cache (compared to the default remote.) It can be used to see what files
   or project. Note that both options can be combined, for example using the
   `-aT` flag.
 
-- `--all-commits` - same as `-a` or `-T` above, but applies to _all_ Git  
-  commits as well as the workspace. Useful for uploading all the data used in
-  the entire existing commit history of the project.
+- `--all-commits` - same as `-a` or `-T` above, but applies to _all_ Git commits
+  as well as the workspace. Useful for downloading all the data used in the
```
Comment (shcheklein) [inline content/docs/command-reference/push.md:75]: same here - why downloading?

File: content/docs/command-reference/params/index.md [hunk 3]
```diff
@@ -159,6 +159,7 @@ params.yaml  train.layers    None   9
 ```
 
 This command shows the difference in parameters between the workspace and the
-last committed version of the `params.yaml` file. In our example, there's no
-previous version, which is why all `Old` values are `None`. See `params diff` to
-learn more about the `diff` command.
+last committed version of the `params.yaml` file. In our example there's no
+previous version, which is why all `Old` values are `None`.
+
+> See `params diff` to learn more about the `diff` command.
```
Comment (shcheklein) [inline content/docs/command-reference/params/index.md:165]: add emoji?

File: content/docs/command-reference/metrics/show.md (hunk 1 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/show.md]: Bad copy paste probably... Thanks. Fixed in 857919e6.

File: content/docs/command-reference/push.md (hunk 2 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/push.md]: Fixed in 857919e6 as well. PTAL (I changed a few other refs for correctness and consistency).

File: content/docs/command-reference/params/index.md (hunk 3 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/params/index.md:165]: If you prefer, why not. Which one though?

But it's a pretty simple note, I wouldn't usually.

Comment (shcheklein) [reply content/docs/command-reference/params/index.md:165]: like 💡 I guess - since it's educational ... `learn more`

**Review by shcheklein — APPROVED** [review]

File: content/docs/command-reference/params/index.md (hunk 3 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/params/index.md]: Yeah could be. But actually this note is redundant now that I think about it, I just removed it 😋

**Pull request merged**